packages/survey-creator-core/src/creator-responsivity-manager.ts

```diff
@@ -4,16 +4,17 @@ export class CreatorResponsivityManager {
   private resizeObserver: ResizeObserver = undefined;
   private currentWidth;
   private screenWidth: { [key: string]: number } = {
-    "xxl": 1920,
-    "xl": 1280,
-    "l": 960,
-    "m": 768,
-    "s": 376,
+    "xxl": 1800,
+    "xl": 1500,
+    "l": 1200,
+    "m": 900,
+    "s": 600,
   }
   private getScreenWidth(): string {
     let res;
     Object.keys(this.screenWidth).forEach((mode: string) => {
-      if (!res && this.container && !!this.screenWidth[mode] && this.container.offsetWidth >= this.screenWidth[mode]) {
+      if (!res && this.container && !!this.screenWidth[mode] && this.container.offsetWidth >= this.screenWidth[mode] 
+        && this.container.offsetWidth >= this.screenWidth[mode]) {
         res = mode;
```

Comment:
remove duplicate "this.container.offsetWidth >= this.screenWidth[mode]"

Reply:
https://github.com/surveyjs/survey-creator/pull/2211/commits/0f6734c973ac434d687257b03d79289629b42e15